wiki/About/vi.md

```diff
@@ -0,0 +1,5 @@
+# About
```

Comment:
translate the title as well

Reply:
Hi TPGPL. That's kinda unneccessary. Every Vietnamese people of us all know that! Even every wiki which translated must keep the "About" text

Reply:
I don't get why you leave a translatable word untranslated. It would feel so wrong looking at the page half translated, half not (this is the header).

Reply:
Ok! Lemme take times to fix it. But some terms in osu is very hard to translate to Vietnamese. Whatever, I'm gonna spend several days doing it 😉

Reply:
fyi you don't need to close a PR to fix it, just push changes to your forked repo. Also there are some terms better let them be in original language (like I did in osu-web).

Reply:
much thanks! I know you are Vietnamese same as me, I will let you in my forked repo.

Reply:
you can as well translate is as `About osu! wiki`